pokedex/pokedex.lua

```diff
@@ -279,16 +305,24 @@ function M.get_sprite(pokemon, variant)
 	if pokemon_index == -1 then
 		return "-1MissingNo", "pokemon0"
 	end
-	local pokemon_sprite = pokemon_index .. pokemon
+	
+	local data = get_pokemon_raw(pokemon)
+
+	local sprite_suffix = pokemon
+	if data.variant_data then
+		if data.variant_data.sprite_suffix then
+			sprite_suffix = data.variant_data.sprite_suffix
+		elseif data.variant_data.variants and data.variant_data.variants[variant] and data.variant_data.variants[variant].original_species then
```

Comment:
This block is used for Arceus, Pumpkaboo, and Gourgeist, who are the only pokemon species that use the same image for all variants.